examples/nlp/dialogue_state_tracking/data/multiwoz/schema.json

```diff
@@ -0,0 +1,636 @@
+[
+  {
+    "service_name": "hotel",
+    "slots": [
+      {
+        "name": "hotel-pricerange",
+        "description": "the price range of the hotel",
+        "possible_values": [
+          "$100",
+          "cheap",
+          "cheap>moderate",
+          "cheap|moderate",
+          "expensive",
+          "moderate"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "hotel-type",
+        "description": "the type of the hotel",
+        "possible_values": [
+          "guesthouse",
+          "hotel",
+          "hotel|guesthouse"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "hotel-parking",
+        "description": "does the hotel have free parking",
+        "possible_values": [
+          "free",
+          "no",
+          "yes"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "hotel-bookday",
+        "description": "the day of hotel booking",
+        "possible_values": [
+          "friday",
+          "friday>tuesday",
+          "monday",
+          "monday<thursday",
+          "saturday",
+          "saturday|tuesday",
+          "sunday",
+          "sunday>monday",
+          "thursday",
+          "tuesday",
+          "wednesday",
+          "wednesday|friday"
+        ],
+        "is_categorical": true
```

Comment:
fascinating... clearly this and the previous lists are not _complete_ in any sense...